src/style.css

```diff
@@ -0,0 +1,128 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #eef;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
`box-sizing` should be applied to all elements, put this style under `*` selector